server/services/licenceService.test.ts

```diff
@@ -41,7 +47,6 @@ describe('Licence Service', () => {
   describe('Create Licence', () => {
     beforeEach(() => {
       prisonerService.getPrisonerDetail.mockResolvedValue({} as PrisonApiPrisoner)
-      communityService.getStaffDetail.mockResolvedValue({} as CommunityApiStaffDetails)
       communityService.getProbationer.mockResolvedValue({ offenderManagers: [] } as OffenderDetail)
```

Comment:
I have a suspicion that we'll need to com's details on the licence whilst it is still in progress, as OMU users (I think) might be getting visibility of IN_PROGRESS licences, and not be able to contact or see the COMs details until is it submitted. One for the team to discuss..

Reply:
I chatted to Jon about this, if it's still in progress we can pull it from delius, but thats out of scope for now